src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -42,8 +42,8 @@ internal partial class CodeFixService : ICodeFixService
         private readonly ImmutableDictionary<LanguageKind, Lazy<ISuppressionFixProvider>> _suppressionProvidersMap;
         private readonly IEnumerable<Lazy<IErrorLoggerService>> _errorLoggers;
 
-        private ImmutableDictionary<CodeFixProvider, FixAllProviderInfo> _fixAllProviderMap;
-
+        private ImmutableDictionary<object, FixAllProviderInfo> _fixAllProviderMap;
```

Comment:
Perhaps add a comment that this is object because it can either be a CodeFixProvider or an ISuppressionFixProvider. I don't like the duality but I guess there is no easy way around this..